src/Display/Display.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const DisplayStyles = styled.div`
```

Comment:
I like your explaination about what Display is and why it's not a header element in the PR :) Lets put that as living documentation in the story, or as a comment here?